scripts/run_examples

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env bash
+
+set -o errexit
+set -o nounset
+
+# Run oak server
+./scripts/build_server_docker
+readonly USERNAME="$(id --user --name)"
+readonly SCRIPTS_DIR="$(dirname "$0")"
+
+docker run \
+  --detach \
```

Comment:
I think this also causes it to print the container id, so ideally you should be able to parse that directly?
Also, IIUC `--detach` is the only difference from the command in the `docker_run` script; if that's correct, could you add a comment here, so it's easier to keep the two in sync going forward?

Reply:
Sure! How about a script `docker_run_detach`?

Reply:
Or make that an option to the `docker_run` script itself.

Reply:
`docker_run --detach`

Reply:
I stick with a comment to keep the commands in sync for now.